tests/common.go

```diff
@@ -614,6 +614,23 @@ func PrintCommandOutput(cmnd string) {
 
 }
 
+func PrintSvPoolStatus(node node.Node) {
+	PrintCommandOutputOnNode("pxctl sv pool show", node)
+}
+func PrintCommandOutputOnNode(cmnd string, n node.Node) {
```

Comment:
How about using `runCmdGetOutput` in `tests/common.go`? Also, could we add a line between these two functions?

Reply:
You are right the runCmdGetOutput is doing samething, hence replaced with the following subroutine